src/combinators/sequence.ts

```diff
@@ -8,6 +8,7 @@ import type { Parser, ToTuple } from '@types'
  * @returns Tuple of values returned by `ps` parsers
  */
 export function sequence<T extends Array<Parser<unknown>>>(...ps: T): Parser<ToTuple<T>>
+export function sequence<T extends Array<Parser<unknown>>>(...ps: T): Parser<ToTupleOrArray<T>>
```

Comment:
In my testing without this we can't infer correct type of `Array` and it will always be `[]` [here](https://github.com/norskeld/sigma/pull/75/files#diff-0a9454b607fdfed0e5d3527f75e1e927b8336ecc81f008cb66b7fbb5f160aafdR16)